README.md

```diff
@@ -63,6 +63,12 @@
     <li>CSS</li>
   </ul>
 </details>
+<details>
+  <summary>API</summary>
+  <ul>
+    <li>Bookstore API</li>
+  </ul>
+</details>
```

Comment:
- [ ] Just as previously suggested, the link to your MIT license in the README should point to the license file in your repository.